bench-vortex/src/bin/tpch.rs

```diff
@@ -55,6 +59,8 @@ struct Args {
     all_metrics: bool,
     #[arg(long)]
     export_spans: bool,
+    #[arg(long, default_value_t = false)]
+    emit_plan: bool,
```

Comment:
Does this work for duckdb?

Reply:
Nope but this is not the only flag which doesn't work for DuckDB. Worthwhile approaching that problem, also for DataFusion, in a separate PR.